assets/js/carousel-slider.js

```diff
@@ -55,14 +58,14 @@ const sliderSk = function () {
       curSlide--;
     }
     goToSlide(curSlide);
-    activateDot(curSlide);
+    //activateDot(curSlide);
   };
 
   const init = function () {
     goToSlide(0);
-    createDots();
+    //createDots();
```

Comment:
Es sollte kein auskommentierter code eingecheckt werden. Wird der noch gebraucht? Warum steht er da? Wurde er versehentlich auskommentiert?

Reply:
Das sind die Punkte, die man benutzen kann um um Carousel zu navigieren. Habe die mal drin gelassen, da man die vielleicht noch in Zukunft integrieren könnte, wenn das Standard Carousel mal läuft